src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java

```diff
@@ -20,7 +21,8 @@
 	
 	default String fetchApiJsonFromApiServer(boolean chatty, TaskListener listener, 
     		Map<String,String> overrides, String apiDomain) {
-		return httpGet(chatty, listener, overrides, this.getApiURL(overrides) + "/swaggerapi/" + apiDomain + "/v1");
+		DefaultClient client = (DefaultClient) this.getClient(listener, getDisplayName(), overrides);
```

Comment:
with my suggestion in `IOpenShiftPlugin` I believe this code path can remain unchanged